plugins/hydra_ray_launcher/tests/ray_test_utils.py

```diff
@@ -0,0 +1,107 @@
+# # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import os
```

Comment:
I would want to go the changes together, I don't like seeing so much code duplication.